src/parser/BlackBearSyntax.C

```diff
@@ -18,7 +18,10 @@ namespace BlackBear
 {
 
 void
-associateSyntax(Syntax & /*syntax*/, ActionFactory & /*action_factory*/)
+associateSyntax(Syntax & syntax, ActionFactory & /*action_factory*/)
 {
+  registerSyntax("EmptyAction", "Constraints/EqualValueEmbeddedConstraint");
+  registerSyntax("EqualValueEmbeddedConstraintAction",
```

Comment:
Registering this as `EqualValueEmbeddedConstraintAction` isn't really consistent with how we name other classes like this. Maybe call it `Multiple` or `Batch` or `Master` instead? I think I like `Batch` the best. Then the syntax would look like: `[Constraints/EqualValueEmbeddedConstraint/Batch]`

Reply:
Corrected!